src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -489,17 +489,24 @@ class FilterEditorUI extends Component<Props, State> {
       const { index, disabled, negate } = this.props.filter.meta;
       const newIndex = index || this.props.indexPatterns[0].id!;
       const body = JSON.parse(queryDsl);
-      const filter = buildCustomFilter(newIndex, body, disabled, negate, alias, $state.store);
+      const filter = buildCustomFilter(
+        newIndex,
+        body,
+        disabled || false,
+        negate || false,
+        alias || null,
+        $state.store
+      );
       this.props.onSubmit(filter);
     } else if (indexPattern && field && operator) {
       const filter = buildFilter(
         indexPattern,
         field,
         operator.type,
         operator.negate,
-        this.props.filter.meta.disabled,
+        this.props.filter.meta.disabled || false,
```

Comment:
I think that these default can go into the function definitions themselves (not sure if in this PR)

Reply:
i'll leave it as it is at the moment